config/crd/kustomization.yaml

```diff
@@ -0,0 +1,19 @@
+# This kustomization.yaml is not intended to be run by itself,
+# since it depends on service name and namespace that are out of this kustomize package.
+# It should be run by config/default
+resources:
+- bases/hydra.ory.sh_oauth2clients.yaml
+# +kubebuilder:scaffold:crdkustomizeresource
+
+patches:
```

Comment:
In the current version of `patches` is `patchesStrategicMerge`, please update field name in all `kustomization.yaml` files

Reply:
regenerated using kubebuilder v2.0.0.